src/storyshots.spec.js

```diff
@@ -5,6 +5,7 @@
  * Therefore, we mimick the Storybook API to get the components and then run the snapshot tests
 */
 let mockStories = []
+const consoleErrorSpy = jest.spyOn(global.console, 'error')
```

Comment:
This should catch `console.log` messages in storyshot tests and let the test fail if one occurs.